compiler/modules/front.syntax/tests/data/testParser.json

```diff
@@ -238,8 +239,6 @@
     ["V", "expr", "a(x).b.c[y]", "((a(x).b).c)[y]"],
     ["V", "expr", "a[b.c]"],
     ["E", "expr", "a.1", ""],
-    { "comment" : "may get allowed under  ballerina-spec#34" },
```

Comment:
@jclark Is there no need for map getting allowed for field access to be handled in the parser?

Reply:
`a.map` is an error.